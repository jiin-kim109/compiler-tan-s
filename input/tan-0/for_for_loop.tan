main {
    for(i from 2 to 5) {
        for (j from 1 to 9) {
            if (i == 3) {
                continue;
            }
            if (j == 8) {
                break;
            }
            print i*j \s;
        }
        print \n ;
    }
    print \n ;
}