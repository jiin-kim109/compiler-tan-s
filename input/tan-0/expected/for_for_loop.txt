2 4 6 8 10 12 14

4 8 12 16 20 24 28
5 10 15 20 25 30 35
